presto-hive/src/test/java/com/facebook/presto/hive/TestHiveExternalWorkersQueries.java

```diff
@@ -228,4 +239,51 @@ public void testValues()
     {
         assertQuery("SELECT 1, 0.24, ceil(4.5), 'A not too short ASCII string'");
     }
+
+    public static void main(String[] args)
```

Comment:
Can we separate out this into another new query runner? Maybe a new class called `HiveExternalWorkerQueryRunner`. Then use this new class for (1) your local laptop development and (2) as a util to be used in this test (`TestHiveExternalWorkersQueries.java`)